Validation/RecoVertex/src/TrackParameterAnalyzer.cc

```diff
@@ -42,10 +42,9 @@ TrackParameterAnalyzer::TrackParameterAnalyzer(const edm::ParameterSet& iConfig)
   , verbose_( iConfig.getUntrackedParameter<bool>( "verbose", false ) ) {
    //now do whatever initialization is needed
   // open output file to store histograms}
-  TString tversion(edm::getReleaseVersion());
-  tversion = tversion.Remove(0,1);
-  tversion = tversion.Remove(tversion.Length()-1,tversion.Length());
```

Comment:
In reality we should probably add a param in `edm::getReleaseVersion()` which default value. So we could get back CMSSW version with quotes and without quotes. I would not expect to get quotes from `edm::getReleaseVersion()` call.